libs/fixtures/data/electionMinimalExhaustiveSample/cvrFiles/cdf_cast_vote_record_report.json

```diff
@@ -0,0 +1,400 @@
+                  "SelectionPosition": [
+                    {
+                      "HasIndication": "yes",
+                      "NumberVotes": 1
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@type": "CVR.CVR",
+      "BallotStyleId": "2F",
+      "BallotStyleUnitId": "precinct-1",
+      "PartyIds": ["1"],
+      "CreatingDeviceId": "scanner-1",
+      "ElectionId": "some-id",
+      "BatchId": "batch-2",
+      "UniqueId": "id-172",
+      "vxBallotType": "absentee",
+      "vxTestBallot": true,
+      "BallotImage": [
+        {
+          "@type": "CVR.ImageData"
+        },
+        {
+          "@type": "CVR.ImageData",
+          "Location": "./ballot-12312321312-back.jpeg"
+        }
+      ],
+      "CurrentSnapshotId": "id-172-original",
+      "CVRSnapshot": [
+        {
+          "@id": "id-172-original",
+          "@type": "CVR.CVRSnapshot",
+          "Type": "original",
+          "CVRContest": [
+            {
+              "@type": "CVR.CVRContest",
+              "ContestId": "best-animal-fish",
+              "CVRContestSelection": [
+                {
+                  "@type": "CVR.CVRContestSelection",
+                  "ContestSelectionId": "salmon",
+                  "SelectionPosition": [
+                    {
+                      "HasIndication": "yes",
+                      "NumberVotes": 1
+                    }
+                  ]
+                }
+              ]
+            },
+            {
+              "@type": "CVR.CVRContest",
+              "ContestId": "aquarium-council-fish",
+              "CVRContestSelection": [
+                {
+                  "@type": "CVR.CVRContestSelection",
+                  "ContestSelectionId": "manta-ray",
+                  "SelectionPosition": [
+                    {
+                      "HasIndication": "yes",
+                      "NumberVotes": 1
+                    }
+                  ]
+                },
+                {
+                  "@type": "CVR.CVRContestSelection",
+                  "ContestSelectionId": "write-in-0",
+                  "SelectionPosition": [
+                    {
+                      "HasIndication": "yes",
+                      "NumberVotes": 1,
+                      "CVRWriteIn": {
+                        "@type": "CVR.CVRWriteIn",
+                        "vxPositionInBallotImage": {
+                          "Side": "back",
+                          "ContestRect": {
+                            "x": 0,
+                            "y": 0,
+                            "width": 800,
+                            "height": 200
+                          },
+                          "WriteInRect": {
+                            "x": 0,
+                            "y": 0,
+                            "width": 800,
+                            "height": 100
+                          }
```

Comment:
We currently include layout information in the CVR to be able to tell where to zoom in in our transcription UI. Currently we include the entire computed layout in the CVR. We have a few options:
- Continue including the entire computed layout and put it here
- Continue including the entire computed layout and put a file reference to it here
- Only include the layout relevant to the contest

The last option is what I lean towards because it's the most specific and compact, even though it's a bit of extra work.

Reply:
cc @jonahkagan @carolinemodic @eventualbuddha

Reply:
I like just moving the logic of this to the scanners instead of VxAdmin, it feels crazy in VxAdmin to be trying to figure out the coordinates

Reply:
It looks like the CDF spec intends for us to just include an image of the write-in here directly. Why customize to include layout info instead of just doing that? Is the goal to reduce the number of images and thus the size of the CVR?

Reply:
For our transcription UX, we use the image of the entire ballot. It's possible that the ballot has multiple write-ins, so I think we should include the image of the ballot at the top-level to and avoid duplication. 

We could include an image of the write-in, but we also would need to include an image of the contest because we use that in our UX as well. So, without changing our UX, we have to somehow add something custom.

Reply:
Gotcha, that makes sense.

A couple more thoughts:
- The contest layout info will be duplicated for each write-in. Not a huge deal but could be normalized.
- Are there times that we would need layout info for anything other than adjudicating write-ins? Are there other issues that might require adjudication? If so, it might make sense to include layout info on the `CVRContest` object and on the `CVRContestSelection` object.

None of these are strong recommendations, just poking and exploring options

Reply:
We include the layout information per CVR because the location of these things in each image is not unique due to rotation, stretching, cropping, etc. from the scanning process.

We could choose to normalize images like with currently do with VX ballots. This would mean that we can use a layout per ballot style, but it has a downside. We either have to interpret using the normalized image (not sure this is a good idea), ship both the normalized and original images with the CVRs, or only use original images with per-ballot layout information.

Reply:
@eventualbuddha I completely understand the rationale.

But looking at the existing code, it looks to me like we're _not_ using the calculated layout and we're using the ballot style layout. Here's where we pass layouts to the cast vote record, and we're querying the ballot style layout metadata saved in the DB:
https://github.com/votingworks/vxsuite/blob/a487926b23ee5dd30ce6e9078927ec230151586e/apps/scan/backend/src/cvrs/export.ts#L122-131

If we _are_ using the calculated layouts, then could you explain what I'm missing. If we actually _aren't_, why is that and does it matter to start using them?